master/job.go

```diff
@@ -0,0 +1,166 @@
+package master
+
+import (
+	"bytes"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+// NOTE: consider using math/big for some of the operations in this package
+
+var charSetSlice [][]byte
+
+func init() {
+	const nums string = "0123456789"
+	const alphaLower string = "abcdefghijklmnopqrstuvwxyz"
+	const alphaMixed string = alphaLower + "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	const alphaNumLower string = nums + alphaLower
+	charSetSlice = [][]byte{
```

Comment:
Maybe it is a better idea to put the constants in the lib, since I also need them for the brute-forcer